File: src/app/models/book.ts
export class Book {
    id: number;
    title: string;
    author: string;
    genre: string;
    
    description: string;
    status : string ;
  
    constructor(id:number ,title: string, author: string, genre: string, description: string, status: string) {
      this.id=id;
      this.title = title;
      this.author = author;
      this.genre = genre;
      
      this.description = description;
      this.status = status;
    }
  }
  

File: BookReadingListAPI/Models/Book.cs
using BookReadingListAPI.Models;

public class Book
{
  public int Id { get; set; }
  public string Title { get; set; }
  public string Description { get; set; }
  public string Status { get; set; }
  public int PublisherId { get; set; }
  public Publisher Publisher { get; set; }
  public int GenreId { get; set; }
  public Genre Genre { get; set; }
  public int AuthorId { get; set; }
  public Author Author { get; set; }
}
